frontend/src/css/modules/web.scss

```diff
@@ -1,6 +1,12 @@
 /*main container style*/
-
+$break-small: 600px;
```

Comment:
Also, instead of using 600px use 992px. We are using the mobile nature upto tablet size.